lib/tree/peer.go

```diff
@@ -0,0 +1,74 @@
+package tree
+
+import (
+	"github.com/dedis/cothority/lib/hashid"
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/crypto/abstract"
+	"strconv"
+)
+
+// Peer represents a Cothority-member
+type Peer struct {
+	// The hostname of the peer
+	Name string
+	// The port-number
+	Port int
```

Comment:
Perhaps better just to use a single string, which can be in "hostname:port" syntax (perhaps with port optional)?  I think the Go network code already supports that syntax, doesn't it?  That would make dependence on the exact structure of endpoint IDs less pervasively hard-coded.